scopus-import/src/main/java/no/sikt/nva/scopus/conversion/CristinPersonRetriever.java

```diff
@@ -0,0 +1,78 @@
+package no.sikt.nva.scopus.conversion;
+
+import static java.util.Objects.nonNull;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import no.scopus.generated.AuthorGroupTp;
+import no.scopus.generated.AuthorTp;
+import no.sikt.nva.scopus.conversion.model.AuthorIdentifiers;
+import no.sikt.nva.scopus.conversion.model.CristinPersonContainer;
+import no.sikt.nva.scopus.conversion.model.cristin.CristinPerson;
+import no.sikt.nva.scopus.paralleliseutils.ParallelizeListProcessing;
+
+public class CristinPersonRetriever {
+
+    private final CristinConnection cristinConnection;
+    private final PiaConnection piaConnection;
+
+    public CristinPersonRetriever(CristinConnection cristinConnection, PiaConnection piaConnection) {
+        this.cristinConnection = cristinConnection;
+        this.piaConnection = piaConnection;
+    }
+
+    public Map<AuthorIdentifiers, CristinPerson> retrieveCristinPersons(List<AuthorGroupTp> authorGroupTps) {
```

Comment:
attempted to isolate all the logic for retrieving cristinPerson to a single class. But naming of this class, particularly when you look at the rest of the class names, is not great.